config/crd/bases/vmoperator.vmware.com_virtualmachinereplicasets.yaml

```diff
@@ -103,11 +103,13 @@ spec:
                           items:
                             type: string
                           type: array
+                          x-kubernetes-list-type: atomic
```

Comment:
@aruneshpa Please note this and the next entry are a result of the dependency update.